primitives/src/lib.rs

```diff
@@ -32,6 +32,9 @@ pub mod locktime;
 pub mod opcodes;
```

Comment:
Oh, looking at the line above, we probably cannot not reexport `serde` here. OK.